### PR TITLE
add wwr variation per RBSA II data

### PR DESCRIPTION
**Pull request by mdahlhausen**
- binned WWRs for multifamily and single family buildings from RBSA II data
- for calculations, see WWR estimates in EULP calibration directory on sharepoint
- Assuming '50 or more Unit' building type is midrise/highrise with default 30% WWR for all buildings

Addresses #[issue number here].

## Pull Request Description

[description here]

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] PAT project measures/outputs have been updated
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing (green) on circleci
- [ ] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: project_multifamily_beta/housing_characteristics/Window Areas.tsv [hunk 1]
```diff
@@ -1,2 +1,10 @@
...
+Dependency=Geometry Building Type ACS	Option=F6 B6 L6 R6	Option=F9 B9 L9 R9	Option=F12 B12 L12 R12	Option=F15 B15 L15 R15	Option=F18 B18 L18 R18	Option=F30 B30 L30 R30
+Mobile Home	0.309	0.210	0.354	0.027	0.096	0.004
+Single-Family Attached	0.309	0.210	0.354	0.027	0.096	0.004
+Single-Family Detached	0.309	0.210	0.354	0.027	0.096	0.004
+2 Unit	0.122	0.063	0.432	0.027	0.249	0.107
+3 or 4 Unit	0.122	0.063	0.432	0.027	0.249	0.107
+5 to 9 Unit	0.122	0.063	0.432	0.027	0.249	0.107
+10 to 19 Unit	0.122	0.063	0.432	0.027	0.249	0.107
+20 to 49 Unit	0.122	0.063	0.432	0.027	0.249	0.107
+50 or more Unit	0.0	0.0	0.0	0.0	0.0	1.0
```
Comment (ejhw) [inline project_multifamily_beta/housing_characteristics/Window Areas.tsv]: We actually already have a breakout of low vs. high for these building types. See https://github.com/NREL/OpenStudio-BuildStock/blob/master/project_multifamily_beta/housing_characteristics/Geometry%20Building%20Number%20Units%20HL.tsv. Please change your dependency to this instead.

File: project_multifamily_beta/housing_characteristics/Window Areas.tsv [hunk 2]
```diff
@@ -1,2 +1,10 @@
-Option=F15 B15 L15 R15
-1.000000
+Dependency=Geometry Building Type ACS	Option=F6 B6 L6 R6	Option=F9 B9 L9 R9	Option=F12 B12 L12 R12	Option=F15 B15 L15 R15	Option=F18 B18 L18 R18	Option=F30 B30 L30 R30
+Mobile Home	0.309	0.210	0.354	0.027	0.096	0.004
+Single-Family Attached	0.309	0.210	0.354	0.027	0.096	0.004
+Single-Family Detached	0.309	0.210	0.354	0.027	0.096	0.004
```
Comment (ejhw) [inline project_multifamily_beta/housing_characteristics/Window Areas.tsv]: How did you get the probabilities for each bin? What bounds did you use for each bin? Looking at your RBSA_wwr.xlsx, I get 15% for 0–0.08 WWR, which is lower than 30.9%.

Also, please add 6 sig figs to all values per our new guidelines: https://resstock.readthedocs.io/en/latest/advanced_tutorial/writing_housing_characteristics.html)

**Comment by joseph-robertson** [comment]
We're also going to want to try, as much as possible, to create tsv files using automated/reproducible methods along with tags describing what data they're based on. See an [example](https://github.com/NREL/OpenStudio-BuildStock/blob/master/data/recs/2009/tsv_maker.py).

File: project_multifamily_beta/housing_characteristics/Window Areas.tsv (hunk 1 above)
Comment (mdahlhausen) [reply project_multifamily_beta/housing_characteristics/Window Areas.tsv]: Fixed with https://github.com/NREL/OpenStudio-BuildStock/pull/393/commits/51ea1d694c4141739025eb355da7392486b47331

File: project_multifamily_beta/housing_characteristics/Window Areas.tsv (hunk 2 above)
Comment (mdahlhausen) [reply project_multifamily_beta/housing_characteristics/Window Areas.tsv]: I'm using the RBSA II data, not the RBSA data.

Using:
[RBSAII_BuildingOneLine.xlsx](https://nrel.sharepoint.com/:x:/r/sites/LoadProfiles/Shared%20Documents/05%20Calibration/4%20Calibration%20and%20Validation/Window%20to%20Wall%20Ratio%20Estimates/RBSAII_BuildingOneLine.xlsx?d=w0c11e1086e6e4ce681d150183fbf6ce3&csf=1&e=ShRGeS)
[RBSAII_One Line Summary](https://nrel.sharepoint.com/:x:/r/sites/LoadProfiles/Shared%20Documents/05%20Calibration/4%20Calibration%20and%20Validation/Window%20to%20Wall%20Ratio%20Estimates/RBSAII_One%20Line%20Summary.xlsx?d=w616859c4fd434e078ba13700360bc5a3&csf=1&e=t019V0)

See the highlighted yellow boxes.  I picked the WWRs to match what was already in the options lookup and to best synchronize between SF and lowrise.  Bins selected so the average in the bin equals the WWR target.  Then calculated the percentage in each bin.

6 figures added with https://github.com/NREL/OpenStudio-BuildStock/pull/393/commits/51ea1d694c4141739025eb355da7392486b47331

Comment (ejhw) [reply project_multifamily_beta/housing_characteristics/Window Areas.tsv]: @joseph-robertson Do you recall if we are supposed to use the "Strata Population Estimate" as a weight for these samples?

Comment (ejhw) [reply project_multifamily_beta/housing_characteristics/Window Areas.tsv]: @mdahlhausen I figured out that we're supposed to use "Site Case Weight" to weight each sample. Can you update your numbers using these weights? 
image
https://neea.org/img/uploads/Residential-Building-Stock-Assessment-II-Single-Family-Homes-Report-2016-2017.pdf

I'm somewhat concerned by the big difference in WWR between RBSA I and RBSA II. Ideally, we'd pool the samples and use both, but I'm fine moving forward with just RBSA II if necessary.

A future work item would be to see if there is a relationship between WWR and vintage.

Comment (mdahlhausen) [reply project_multifamily_beta/housing_characteristics/Window Areas.tsv]: @ejhw is the idea that each site ID fits into one of these strata, and I should weight each building accordingly? That will mean I'll need to create a custom averaging function which will take some time.  Do you have an example of this done for some other characteristic?

Comment (ejhw) [reply project_multifamily_beta/housing_characteristics/Window Areas.tsv]: Right. It's possible it won't change the distro much, but good to check at least. An example of that from RECS is here: https://github.com/NREL/resstock-estimation/blob/master/utils/tsv_maker.py#L73

You can also do it in Tableau by making a histogram and then switching SUM([Site Case Weight]) for CNT(...).

**Comment by asparke2** [comment]
Moved to https://github.com/NREL/OpenStudio-BuildStock/pull/412 to trigger CI

**Pull request closed without merging**